tiff-2-png-converter.py

```diff
@@ -7,100 +7,249 @@
-    gamma = False
-    gamma_val = 1
-    num_imgs = None
-    source = None
-    dest = None
-
-    @staticmethod
-    def add_arguments(parser):
-        parser.add_argument(
-            "-s",
-            "--source",
-            required=True,
-            help="Path to directory where the image sequence is located",
-        )
-        parser.add_argument(
-            "-d",
-            "--dest",
-            required=True,
-            help="Path to the directory and filename of the result merged image (png by default) will be saved. ",
-        )
-        parser.add_argument(
-            "--verbose", help="Generate messages ouput ", action="store_true"
-        )
-
-        parser.add_argument("--num_imgs", help="Generate messages ouput ")
-
-        parser.add_argument(
-            "--equalize1",
-            help="equilize histogram in volume slices using opencv functionality",
-            action="store_true",
-        )
-
-        parser.add_argument(
-            "--equalize2",
-            help="equilize histogram in volume slices using custom implementation",
-            action="store_true",
-        )
-
-        parser.add_argument("--gamma", help="Apply gamma correction")
-
-    @staticmethod
-    def configure(args):
-
-        name, extension = os.path.splitext(args.dest)
-        if os.path.exists(args.source) and os.path.exists(os.path.dirname(name)):
-            TiffProcessor.source = args.source
-            TiffProcessor.dest = name
+def interpolate_data(img_array, width, heigth, num_images):
+    ## STILL IN DEVELOP - NOT SURE IF WE NEED IT
+    # ptython list to numpy 3d array
+    img_3d_array = np.array((width, heigth))
+    for z in range(num_images):
+        img_2d_array = img_array[z]
+        if z == 0:
+            img_3d_array = img_2d_array
+        elif img_2d_array is not None:
+            img_3d_array = np.dstack([img_3d_array, img_2d_array])
         else:
-            raise ValueError("ERROR: Verify source and destination paths exists")
-
-        if args.verbose:
-            TiffProcessor.verbose = True
-        if args.equalize1 or args.equalize2:
-            TiffProcessor.equalize_histogram = True
-            if args.equalize1:
-                TiffProcessor.equalize_histogram_method = (
-                    iph.equalize_image_histogram_opencv
-                )
-            if args.equalize2:
-                TiffProcessor.equalize_histogram_method = (
-                    iph.equalize_image_histogram_custom
-                )
+            img_3d_array = np.dstack([img_3d_array, np.zeros((width, heigth))])
+    min_data = np.amin(np.amin(img_3d_array, axis=2))
+    max_data = np.amax(np.amax(img_3d_array, axis=2))
+    logging.info("Minimum value in the whole array:%d" % (min_data))
+    logging.info("Maximum value in the whole array:%d" % (max_data))
+    min_max_scale = np.arange(min_data, max_data)
+
+
+def merge_rgb(image_channels, rgb_images, width, height, bits_per_sample):
```

Comment:
Are we still going to merge into rgb? I thought we were not going to do that anymore?